rust/Cargo.toml.in

```diff
@@ -2,6 +2,7 @@
 name = "suricata"
 version = "@PACKAGE_VERSION@"
 edition = "2018"
+rust-version = "1.41.1"
```

Comment:
In https://doc.rust-lang.org/nightly/cargo/reference/manifest.html#the-rust-version-field, I read `The first version of Cargo that supports this field was released with Rust 1.56.0. In older releases, the field will be ignored, and Cargo will display a warning.`
Q: Is it correct to try and use it to check for older versions?

Reply:
yeah the warning is expected for 1.41, but harmless

Reply:
the field is used here https://github.com/OISF/suricata/pull/10674/files#diff-a76cf7978f0a981f911e8d68d2351a72a268977304612226433df4fb8203b06fR1231